packages/beacon-node/src/chain/regen/interface.ts

```diff
@@ -30,6 +30,10 @@ export enum RegenFnName {
 
 export type StateCloneOpts = {
```

Comment:
rename `StateCloneOpts` to something else as `asyncShufflingCalculation` is not related to clone

Reply:
updated